src/components/BookmarkCard/BookmarkCardIcon/index.tsx

```diff
@@ -6,12 +6,14 @@ import './index.scss';
 type BookmarkCardIconProps = {
   className?: string;
   black?: boolean;
+  lightgrey?: boolean;
```

Comment:
Rather than separate `black` and `lightgrey` props, should we maybe have a single optional color prop? Something like
```
type BookmarkCardIconProps = {
  className?: string;
  color?: 'black' | 'lightgrey';
  size: 'sm' | 'md' | 'lg';
  onClick?: React.MouseEventHandler<HTMLButtonElement>;
}

// inside BookmarkCardIcon definition
const classes = classnames(
   // other classes
  {
    lightgrey: color === 'black'
  },
  {
    black: color === 'lightgrey'
  },
  // other classes
);
```
This way, we don't have to worry about a weird/invalid state if a caller tries to set both the `black` and `lightgrey` props.

Reply:
Yep, I like that.  Can do 👍